src/internal/connector/exchange/exchange_data_collection.go

```diff
@@ -182,8 +182,7 @@ func (col *Collection) streamItems(ctx context.Context, errs *fault.Bus) {
 		colProgress, closer = observe.CollectionProgress(
 			ctx,
 			col.fullPath.Category().String(),
-			// TODO(keepers): conceal compliance in path, drop Hide()
-			clues.Hide(col.fullPath.Folder(false)))
+			col.LocationPath().Elements())
```

Comment:
@ryanfkeepers is this the correct way to do logging for a `path.Builder`?

Reply:
Yup.  Builders and elements already comply with clues.Concealer.